ptcg_cockatrice.xml

```diff
@@ -274,8 +436,8 @@
 			<type>Pokémon - Poison Ground</type>
 			<pt>4/7</pt>
 			<tablerow>2</tablerow>
-			<text>3BB: Evolve from Nidorina
-				If a Pokémon you control with no counters on it would be put into a graveyard from the battlefield, you may exile it instead. If you do, return it to the battlefield under its owner's control at the beginning of the next end step. Put a +1/+1 counter on it.</text>
+			<text>{3}{B}{B}: Evolve from Nidorina
+If a Pokémon you control with no counters on it would be put into a graveyard from the battlefield, you may exile it instead. If you do, return it to the battlefield under its owner's control at the beginning of the next end step with a +1/+1 counter on it.</text>
```

Comment:
```suggestion
If a Pokémon you control with no counters on it would be put into a graveyard from the battlefield, you may exile it instead. If you do, return it to the battlefield under its owner's control at the beginning of the next end step with an additional +1/+1 counter on it.</text>
```